Helper/HttpRequest.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Linq.Expressions;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using PCL.Core.Utils;
+using PCL.Core.Utils.Net;
+
+namespace PCL.Core.Helper;
+
+public static class HttpRequest
+{
+
+    public static async Task<HttpResponseMessage> GetServerResponse(HttpRequestOptions options)
+    {
+        Exception? lastException = null;
+        while (options.Retry >0)
+        {
+            try
+            {
+                using (HttpRequestMessage request = options.GetRequestMessage())
+                {
+                    using (CancellationTokenSource cts = new CancellationTokenSource(options.Timeout))
+                    {
+                        return await HttpClientManager.GetClient()
+                            .SendAsync(request, HttpCompletionOption.ResponseHeadersRead, cts.Token);
+                    }
+                }

```

Comment:
如果你用 Rider，它会告诉你这个语法糖...
```suggestion
                using (HttpRequestMessage request = options.GetRequestMessage());
                using (CancellationTokenSource cts = new CancellationTokenSource(options.Timeout));
                return await HttpClientManager.GetClient()
                    .SendAsync(request, HttpCompletionOption.ResponseHeadersRead, cts.Token);
```